### ISSUE TITLE
I don't know how to use this code. I need a tutorial.

### ISSUE THREAD

**Issue by lMoraxl** [NTUYWANG103/APEX_AIMBOT#52]
I have no idea about this. I don't know how to create an environment or what software is needed. I hope I can have a tutorial, a detailed tutorial. Preferably in Chinese, thank you.

**Comment by Thiece** [comment]
You'll need basic knowledge of how to use python, and some knowledge of installing in a CUDA environment, so if you can't get started with that at all, consider taking it step by step with the help of ChatGPT.

**Comment by lMoraxl** [comment]
Do I need to download Yolov7? How should I run them?

**Comment by lMoraxl** [comment]
> 您需要有关如何使用 python 的基本知识，以及在 CUDA 环境中安装的一些知识，因此，如果您根本无法开始使用，请考虑在 ChatGPT 的帮助下逐步完成。

Now that the environments have been created, how do I run them next? Please give me a little help

**Comment by Thiece** [comment]
If you make sure the cuda environment and python environment are complete, jump to the file storage directory in python and run apex.py

**Comment by lMoraxl** [comment]
The other environment is ready, except that cv2 library cannot be installed, I am prompted that I cannot connect to the agent, the card is up to now, I need help, neither pip nor conda download can be completed, and cv2 cannot be imported
![QQ截图20230514190025](https://github.com/NTUYWANG103/APEX_AIMBOT/assets/133212927/ebfaa692-fa81-4264-b636-6d8e6efa9fe2)
![Uploading QQ截图20230514190025.png…]()

**Comment by Thiece** [comment]
This looks like a network problem, you can try using a proxy in the terminal

**Comment by lMoraxl** [comment]
It may be the country lockout I am in... The package could not be downloaded.

**Comment by anwar125** [comment]
I uploaded a new exe version here:
shorturl.at/ikvN3

**Comment by ANdeK1314** [comment]
bro ，i cant find out why i got this，can i get some help
![image](https://github.com/NTUYWANG103/APEX_AIMBOT/assets/96864686/e9a10167-4d17-4bff-b14b-639f5792d392)

**Comment by modu82** [comment]
> bro ，i cant find out why i got this，can i get some help ![image](https://user-images.githubusercontent.com/96864686/239668419-e9a10167-4d17-4bff-b14b-639f5792d392.png)

cuda path error，you need find this file ->nvinfer.dll

**Comment by modu82** [comment]
> It may be the country lockout I am in... The package could not be downloaded.

换清华源

**Comment by tytdoge** [comment]
the nvinfer.dll is missing in my cuda file, reinstallation does nor help at all

**Comment by modu82** [comment]
> You'll need basic knowledge of how to use python, and some knowledge of installing in a CUDA environment, so if you can't get started with that at all, consider taking it step by step with the help of ChatGPT.

bro，i need help,,i try change utils/InferenceEngine.py ,I tried to import a self-made model, but there was an error here

**Comment by modu82** [comment]
> the nvinfer.dll is missing in my cuda file, reinstallation does nor help at all

Not just Cuda, have you configured Tensorrt and Cudnn

**Comment by tytdoge** [comment]
yup, i am able to build up the module now, i have copied all the dll and lib manually to the cuda files

**Comment by Lemonsm** [comment]
I strongly recommend that you use Aconda to do it in a virtual environment. Because if you install the environment locally, you will have a headache if there are problems with dependencies.
Your problems are basically caused by the inconsistent version of pytorch-cuda-cudnn. It is recommended that you learn how to correctly set up a training environment. (pytorch-cuda-cudnn recommends using a slightly older version to ensure that their versions are consistent with each other. In addition, the most important thing is that the pycharm compiler must not use the latest one. There will be various problems in the latest one. It is recommended around March 2021/2022 .)

**Comment by w18221** [comment]
> After two weeks of trying, I still can't run them successfully, is there an exe to make it easy for me to run them.

给你个关键词吧，google or baidu 搜索 “北极星 yolo ai”，是exe一键运行的，但是这个软件一般在一些论坛里面，需要花钱购买邀请码来下载

**Comment by NS7IQ** [comment]
![image](https://github.com/NTUYWANG103/APEX_AIMBOT/assets/162043401/68ba7af8-c915-4404-9abc-dca0733f594e)
 
how to fix
